cmd/fixenv.go

```diff
@@ -0,0 +1,700 @@
+/*
+ * Copyright 2018-2019 The NATS Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/nats-io/jwt"
+	"github.com/nats-io/nkeys"
+	"github.com/nats-io/nsc/cmd/store"
+	"github.com/nats-io/nuid"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	GetRootCmd().AddCommand(createFixCmd())
+}
+
+type FixCmd struct {
+	in        []string
+	out       string
+	creds     bool
+	operators int
+	accounts  int
+	users     int
+	nkeys     int
+
+	Keys              map[string]string
+	KeyToPrincipalKey map[string]string
+	Operators         map[string]*OT
+}
+
+type OT struct {
+	OC         jwt.OperatorClaims
+	Jwts       map[string]string
+	Accounts   map[string]*jwt.AccountClaims
+	ActToUsers map[string]jwt.StringList
+}
+
+func NewOT() *OT {
+	var ot OT
+	ot.Jwts = make(map[string]string)
+	ot.Accounts = make(map[string]*jwt.AccountClaims)
+	ot.ActToUsers = make(map[string]jwt.StringList)
+	return &ot
+}
+
+func createFixCmd() *cobra.Command {
+	var params FixCmd
+
+	var cmd = &cobra.Command{
+		Use:   "fix",
+		Short: "rebuilds a project tree from screwed the contents of one or more directories",
```

Comment:
Don't think there should be `screwed` in the description..maybe: "from corrupted contents on one or more directories"?

Reply:
lol yes